pom.xml

```diff
@@ -317,6 +326,9 @@
               <goals>
                 <goal>jar</goal>
               </goals>
+	      <configuration>
+		<additionalparam>${javadoc.doclint.param}</additionalparam>
+	      </configuration>
```

Comment:
First try in development, I had omitted this piece.
Adapted from Esri/geometry-api-java#120